modules/cooler/dump/main.nf

```diff
@@ -20,6 +20,7 @@ process COOLER_DUMP {
 
     input:
     tuple val(meta), path(cool)
+    val resolution
```

Comment:
I don't know the tool well enough, but is there a good reason to pass `resolution` as an input channel rather than via `options.args`?